wasmer/interface.go

```diff
@@ -21,7 +21,7 @@ type InstanceHandler interface {
 	IsFunctionImported(name string) bool
 	IsInterfaceNil() bool
 	Reset() bool
-	Id() string
+	ID() string
```

Comment:
From linter:
```
Check failure on line 43 in mock/world/builtinFunctionsWrapper.go

GitHub Actions
/ test (1.17.6, ubuntu-latest)
cannot use &mock.EnableEpochsHandlerStub{...} (type *mock.EnableEpochsHandlerStub) as type vmcommon.EnableEpochsHandler in field value:
```
and
```
Check failure on line 43 in mock/world/builtinFunctionsWrapper.go

GitHub Actions
/ Build (ubuntu-latest)
cannot use &mock.EnableEpochsHandlerStub{…} (value of type *mock.EnableEpochsHandlerStub) as type vmcommon.EnableEpochsHandler in struct literal:
```
I think I've made the integrations in another PR that is pending being merged in the master branch.

Reply:
It's not the linter, it's the compiler... This happens because I bumped the `elrond-vm-common` version to a newer one that added the `IsWipeSingleNFTLiquidityDecreaseEnabled` method to the `EnableEpochsHandler` interface, while this stub doesn't have it.

Added missing method to the stub.

Reply:
Also, merged the `rc/v1.5.0` branch which contains your linting fixes.